cluster/src/assembly/resources/sbin/start-node.sh

```diff
@@ -18,6 +18,8 @@
 # under the License.
 #
 
+# Set max number of open files
+ulimit -n 65535
```

Comment:
What happens if the current user is not `root`?

Reply:
some suggestions but not blocking:
1. check whether the current user is `root`
2. check if the current `ulimit` is greater than `65535`
3. Add obvious hints that require manual modification